code/web/src/modules/user/api/actions.js

```diff
@@ -82,6 +84,20 @@ export function register(userDetails) {
   }
 }
 
+export function editProfile(user) {
+  return dispatch => {
+    return axios.post(routeApi, mutation({
+      operation: 'userUpdate',
+      variables: user,
+      fields: ['id', 'image']
+    }))
+    dispatch({
+      type: EDIT_PROFILE,
+      user
+    })
+  }
+}
```

Comment:
It definitely looks like the dispatch was needed below the axios function instead of above like how we had it.